debian/ideascube.install

```diff
@@ -0,0 +1,10 @@
+extras/nginx/ideascube etc/nginx/sites-available
```

Comment:
Because it's not needed.

Advice I received from a Debian Developer friend was that what mattered the most was to be consistent: either never use them or always.

On top of that his general feeling was that since they aren't needed, Debian Developers tend to prefer not putting them in even though there might not be a strong consensus on this.

So I decided to trust him and remove them everywhere.